.pipelines/templates/.builder-Marketplace-win-template.yaml

```diff
@@ -24,17 +24,12 @@ jobs:
       -e PUBLISHER=${PUBLISHER} \
       -e OFFER=${OFFER} \
       -e CONTAINER_RUNTIME=${CONTAINER_RUNTIME} \
-      ${DEIS_GO_DEV_IMAGE} ./vhdbuilder/publish/Marketplace/new-windows-sku.sh
+      ${DEIS_GO_DEV_IMAGE} ./vhdbuilder/publish/Marketplace/new-sku-and-add-image-version.sh
     displayName: Create a new marketplace SKU
-  - task: PublishPipelineArtifact@1
```

Comment:
sku-publishing-info.json is used for pushing images, as we have combined sku and image publishing together, we don't have to push this artifact. we log the content of sku-publishing-info.json instead.